### PR TITLE
Add changes to profile

### PR DESCRIPTION
**Pull request by PraneethV-cmd**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -50,10 +50,11 @@ If you wish to work in Big Tech companies handling large-scale issues, it's para
...
+> Rule 2 exists to encourage people to improve their productivity and expand their boundaries. You are still allowed to join the study group and work with VS Code. I wanna add in the **No Mouse** rule too in order to encourage productivity, but I am still working on mastering it myself. But hey, no pressure! We'll figure it out together as we journey towards becoming the best versions of ourselves by exploring new tools and techniques.  Please read the website and decide for yourself like an adult if the group's philosophy is for you or not. You are more than welcome to steal our ideas for yourself without necessarily participating. Any of our members would be more than happy to help, please remember not to be a jerk. These " Ground Rules " are not set in stone and just act as a deterrent to weed out people, that's all.
```
Comment (2SpaceMasterRace) [inline profile/README.md:57]: Nope the wiki isn't gonna sound AI

File: profile/README.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ This study group is about the internals of compilers, programming languages, dis
 BSOD is a one-week jam where you implement a cool piece of software from scratch. Why build things from scratch? Because that's the only way real progress is made. Every impactful project started life as a small experiment. Plus, writing something from scratch turns theory into practice. You may think you know how a piece of technology works, but until you write it yourself, you truly wouldn't understand it completely.
 
 #### [x] OCaml My Caml :
-it's a week-long hacking retreat to work on a variety of OCaml projects, including but not limited to the compiler, concurrency libraries, platform tooling, and benchmarking. Dive into functional programming and learn to appreciate actual ML languages instead of drooling over whitepaper like Haskell ( We will soon be picking apart GHC btw).
+It's a week-long hacking retreat to work on a variety of OCaml projects, including but not limited to the compiler, concurrency libraries, platform tooling, and benchmarking. Dive into functional programming and learn to appreciate actual ML languages instead of drooling over whitepaper like Haskell ( We will soon be picking apart GHC btw).
```
Comment (2SpaceMasterRace) [inline profile/README.md:30]: a short fix, i can take care thanks !

**Review by 2SpaceMasterRace — COMMENTED** [review]
Thanks for the fix, mucho gracias

File: profile/README.md (hunk 1 above)
Comment (PraneethV-cmd) [reply profile/README.md:57]: Brother, i wrote it :(

**Pull request closed without merging**